pkg/tempopb/tempo.proto

```diff
@@ -37,5 +38,5 @@ message PushResponse {
 
 message PushBytesRequest {
   // pre-serialized PushRequests
-  repeated bytes requests = 1;
```

Comment:
don't replace requests, mark it deprecated and add a new proto field.  distributors always use the new field, ingesters use whichever one they happen to get.

we will have to rollout our ingesters first to support both fields and then follow it up with distributors.

Reply:
Nevermind, I understand now.